src/data/uma-title.csv

```diff
@@ -17,10 +17,10 @@
 "[Maverick]","[Maverick]"
 "[ロード・オブ・エンペラー]","[Road of Emperor]"
 "[エンプレスロード]","[Empress Road]"
-"[クエルクス・キウィーリス]",""
+"[クエルクス・キウィーリス]","[Quercus Kiwiris]"
```

Comment:
We talked about this and there's a better version of this but I can't remember

Reply:
i dont even know what this one translate to, but let me know

Reply:
```suggestion
"[クエルクス・キウィーリス]","[Quercus Civilis]"
```

Reply:
> Quercus Civilis A Crown of Oak-branches, given to him that saved a Citizen

Credits to a certain discord user in uma discord